cypress/e2e/smoke/load_safe.cy.js

```diff
@@ -16,14 +16,16 @@ const OWNER_ADDRESS = '0xE297437d6b53890cbf004e401F3acc67c8b39665'
 
 describe('Load existing Safe', () => {
   before(() => {
+    cy.connectE2EWallet()
```

Comment:
Does this test need the connected wallet?

Reply:
Oh! Now we do of course!

Reply:
It doesn't seem to work on the pipeline though. Might need to add a timeout there.